src/instrumentation/historian.ts

```diff
@@ -20,9 +20,14 @@ export default class Historian {
 
     constructor() {
         if (typeof window === 'object') {
-            if (!window.onerror) {
-                window.onerror = this.onerror.bind(this);
-            }
+            let self = this;
+            let oldHandler = window.onerror;
+            window.onerror = function() {
+                if (oldHandler) {
+                    oldHandler.apply(this, arguments);
```

Comment:
Actually I am not sure - this is old handler which expects to be called in window.onerror context...